ciacco/miners/mailquotalimit-list

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/perl
+
+#
+# Copyright (C) 2021 Nethesis S.r.l.
```

Comment:
```suggestion
# Copyright (C) 2022 Nethesis S.r.l.
```

Reply:
I still need a bit of time to understand the year change, sorry